builtin/providers/aws/resource_aws_lambda_event_source_mapping_test.go

```diff
@@ -52,6 +52,29 @@ func TestAccAWSLambdaEventSourceMapping_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSLambdaEventSourceMapping_importBasic(t *testing.T) {
+	resourceName := "aws_lambda_event_source_mapping.lambda_event_source_mapping_test"
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLambdaEventSourceMappingDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSLambdaEventSourceMappingConfig(rInt),
+			},
+
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"enabled", "function_name", "starting_position"},
```

Comment:
That's sad 😢  but can't we deduce `function_name` from `FunctionArn` which the API does return?